detd/manager.py

```diff
@@ -221,6 +234,72 @@ def update_base_time(self, config):
         safety_margin = multiple * period
 
         config.stream.base_time = (now + ns_until_next_cycle) + safety_margin
+    
+    def add_listener(self, config):
+            '''
+            Performs the local configuration for the configuration provided
+            and returns the associated VLAN interface and socket priority
+            Parameters:
+                config: configuration
+            Returns:
+                VLAN interface
+                socket priority
+            '''
+
+            logger.info("Adding listener to InterfaceManager")
+
+
+            # Retrieve device rate
+            try:
+                rate = self.interface.rate
+            except RuntimeError:
+                logger.exception("Error while retrieving device rate")
+                raise
+
+
+            # Assign resources
+            soprio, tc, queue = self.mapping.assign_and_map(config.stream.pcp, self.scheduler.traffics)
+
+            traffic = Traffic(rate, TrafficType.SCHEDULED, config)
+            traffic.tc = tc
+
+
+            # Add stream to schedule
+            try:
+                self.scheduler.add(traffic)
+            except Exception as ex:
+                logger.exception(f"Error while adding traffic to schedule:\n{self.scheduler.schedule}")
+                self.mapping.unmap_and_free(soprio, traffic.tc, queue)
+                raise
+
+
+            # Make sure that the target device is able to implement the resulting schedule
```

Comment:
As we do not support the Schedule concept on Rx, we can safely remove this feasibility check.